R/Task_classif_ecuador.R

```diff
@@ -23,8 +23,9 @@
 NULL
 
 load_task_ecuador = function(id = "ecuador") {
-  b = as_data_backend(readRDS(system.file("extdata", "ecuador.rda",
-    package = "mlr3spatiotemporal")))
+  data = readRDS(system.file("extdata", "ecuador.rda", package = "mlr3spatiotemporal"))
+  rownames(data) = NULL # TODO: better re-save the data set without rownames
```

Comment:
Ahh, that's why we probably had so many troubles dealing with the row_ids...

Reply:
Should also work with rownames, but it is more efficient with integer.